setup.cfg

```diff
@@ -51,9 +51,9 @@ install_requires =
     keyring>=15,<23.5
     keyrings.alt==3.1; sys_platform == "linux"
     deprecated>=1.2.4,<2.0
-    opentelemetry-api~=1.20.0
-    opentelemetry-sdk~=1.20.0
-    opentelemetry-exporter-otlp-proto-http~=1.20.0
+    opentelemetry-api>=1.20.0
```

Comment:
`>`?  Should we lock create a `<` so that we don't run into potential dependency hell?

Reply:
I am going to go back to `~=1.21.0`